src/Microsoft.ML.AutoML/Experiment/Runners/CrossValSummaryRunner.cs

```diff
@@ -157,13 +157,24 @@ private static double GetAverageOfNonNaNScores(IEnumerable<double> results)
             return newResults.Average(r => r);
         }
 
+        /// <summary>
+        /// return the index of value from <paramref name="values"/> that closest to <paramref name="average"/>. If <paramref name="average"/> is non, +/- inf, the first, max/min value's index will be return.
+        /// </summary>
         private static int GetIndexClosestToAverage(IEnumerable<double> values, double average)
         {
             // Average will be NaN iff all values are NaN.
             // Return the first index in this case.
             if (double.IsNaN(average))
                 return 0;
 
+            // Return the max value's index if average is positive inf.
+            if (double.IsPositiveInfinity(average))
+                return values.ToList().IndexOf(values.Max());
+
+            // Return the max value's index if average is positive inf.
```

Comment:
Nit: The comment is the same as in the max case, it should say min and "negative inf"